lib/active_model/serializer.rb

```diff
@@ -177,6 +177,10 @@ def json_key
       end
     end
 
+    def as_json
+      self.class.adapter.new(self).as_json
```

Comment:
@JustinAiken I'm interested in your thoughts on #954 from a usability perspective ( And I we can update and move https://github.com/rails-api/active_model_serializers/pull/967 into the wiki, as @joaomdmoura would prefer )